sled-agent/src/services.rs

```diff
@@ -1320,6 +1317,7 @@ impl ServiceManager {
         filesystems: &[zone::Fs],
         data_links: &[String],
     ) -> Result<RunningZone, Error> {
+        // TODO: ^^^^^^ return a local wrapper over RunningZone
```

Comment:
Is this for this PR?

Reply:
Ah yeah thanks -- I was planning on doing this at first but decided to take a different approach in the PR. Removed the comment.